session/bootstrap.go

```diff
@@ -966,7 +961,8 @@ func upgradeToVer40(s Session) {
 }
 
 func upgradeToVer41(s Session) {
-	doReentrantDDL(s, "ALTER TABLE mysql.user CHANGE COLUMN `Password` `authentication_string` TEXT", infoschema.ErrColumnNotExists)
+	doReentrantDDL(s, "ALTER TABLE mysql.user MODIFY COLUMN `password` TEXT")
+	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN `authentication_string` TEXT as (`password`)", infoschema.ErrColumnExists)
```

Comment:
We should make `password` as a generated column, then we can remove the `password` in the next major release version.